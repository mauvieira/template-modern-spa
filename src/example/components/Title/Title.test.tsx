import { render, screen } from "@testing-library/react";
import { Title } from "./Title";

describe("Title component", () => {
  it("renders correctly", () => {
    const { container } = render(<Title />);

    const component = screen.getByRole('heading', {
      name: /title/i
    });

    expect(container).toMatchSnapshot();
    expect(component).toBeInTheDocument();
  });
});
