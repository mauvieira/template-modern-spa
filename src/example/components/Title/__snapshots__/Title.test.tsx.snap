// Vitest Snapshot v1

exports[`Title component > renders correctly 1`] = `
<div>
  <h1>
    Title
  </h1>
</div>
`;
